docs/guides/getting-started-java-spring.md

```diff
@@ -72,7 +72,7 @@ Next, we'll create a BPMN diagram to represent the transaction model shown at th
 
 1. Open Desktop Modeler.
 2. Click **Create a new diagram** in Camunda 8.
-3. In the properties panel, under the **General** section:
+3. Toggle the properties panel from the Window menu if it is not already visible, and make the following changes in the **General** section:
```

Comment:
I struggled to find the properties panel with a fresh install of Modeler 😅

Reply:
Can you take this feedback to the Desktop Modeler team? I'm not sure if it's collapsed by default.

Reply:
I'm unclear what the feedback is. My experience with a fresh installation of Modeler is that _it is_ collapsed by default. Do you mean that it shouldn't be?

Reply:
I do not love this new UI -- the light gray on white is tough to clearly see.

Perhaps "If the properties panel is not already visible, expand it by clicking the gray toggle on the right side of the screen(or canvas?)."

Reply:
Wow, I didn't even _see_ that light gray affordance until you said something. 

Do you have a strong preference for guiding them to the visual toggle vs. selecting something from the menu as my changes describe? My preference is the menu, but if you feel strongly that it should be about the visual toggle, I will change it.

Reply:
The menu is fine by me!